changes/issue-15551-warn-long-migration

```diff
@@ -0,0 +1 @@
+* **NOTE**: database migrations for this release may take a few minutes to complete depending mostly on how many entries exist in the `software` table.
```

Comment:
@noahtalerman FYI as discussed in https://fleetdm.slack.com/archives/C019WG4GH0A/p1702483968774239?thread_ts=1702415824.628909&cid=C019WG4GH0A, we'll add that note to the changelog to warn users that this release requires a database migration that may take a few minutes.